### PR TITLE
fixed compilation errors

### PR DESCRIPTION
**Pull request by sisuani**
Errors compiling on gentoo and Ubuntu.

### CODE REVIEW

File: src/flowlayout.cpp [hunk 1]
```diff
@@ -38,7 +38,11 @@
 **
 ****************************************************************************/
 
+#if (QT_VERSION < 0x050000)
```
Comment (rudilee) [inline src/flowlayout.cpp:41]: I'm forgot that in readme I mention the Qt min. requirement is 4.6, but currently I'm developing at > 5, silly me :stuck_out_tongue:

**Comment by sisuani** [comment]
Thank you :+1:

**Comment by botunjac** [comment]
Rudi,  thanks on your work.
Regards

Zoran Živanović
 20.01.2015. 16.27, "Samuel Isuani" notifications@github.com је
написао/ла:

**Comment by rudilee** [comment]
No @sisuani , I'm the who should thank you :smile: 

On Tue, Jan 20, 2015 at 10:26 PM, Samuel Isuani notifications@github.com
wrote:

> Thank you [image: :+1:]
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/rudilee/ACM/pull/1#issuecomment-70670951.

## 

Best regards,

Rudi Lee

**Pull request merged**